src/backend/prisma/schema.prisma

```diff
@@ -10,29 +7,122 @@ generator client {
+  first_name        String
+  last_name         String
+  email_id          String             @unique
+  last_login        DateTime
+  role              Int
+  fab_weld_requests Fab_Weld_Request[]
+  change_requests   Change_Request[]
+  changes           Change[]
+  wbs_el_led        WBS_Element[]
+  projects_managed  Project[]
+}
+
+model Fab_Weld_Request {
+  fw_id            Int         @id @default(autoincrement())
+  date_submitted   DateTime
+  part_name        String
+  quantity         Int
+  resources_needed String
+  links            String
+  date_completed   DateTime
+  requestor        User        @relation(fields: [requestor_id], references: [user_id])
+  requestor_id     Int
+  wbs_el           WBS_Element @relation(fields: [wbs_el_id], references: [wbs_el_id])
+  wbs_el_id        Int
+}
+
+model Change_Request {
+  cr_id           Int         @id @default(autoincrement())
+  date_submitted  DateTime
+  type            CR_Type
+  what            String
+  why             String
+  timeline_impact DateTime
+  budget_impact   Int
+  scope_impact    String
+  docs            String
+  date_reviewed   DateTime
+  accepted        Boolean
+  changes         Change[]
+  requestor       User        @relation(fields: [requestor_id], references: [user_id])
+  requestor_id    Int
+  wbs_el          WBS_Element @relation(fields: [wbs_el_id], references: [wbs_el_id])
+  wbs_el_id       Int
+}
+
+model Change {
+  change_id        Int            @id @default(autoincrement())
+  attribute        String
+  original         String
+  new              Boolean
+  date_implemented DateTime
+  implementor      User           @relation(fields: [implementor_id], references: [user_id])
+  implementor_id   Int
+  change_request   Change_Request @relation(fields: [cr_id], references: [cr_id])
+  cr_id            Int
+}
+
+model WBS_Element {
+  wbs_el_id         Int                @id @default(autoincrement())
+  date_created      DateTime
+  wbs_num           String
+  wbs_el_name       String
+  wbs_el_status     WBS_El_Status
+  fab_weld_requests Fab_Weld_Request[]
+  change_requests   Change_Request[]
+  wbs_el_lead       User               @relation(fields: [wbs_el_lead_id], references: [user_id])
+  wbs_el_lead_id    Int
+}
+
+model Project {
+  project_id         Int            @id
+  g_drive_folder     String
+  slide_deck_link    String
+  bom_link           String
+  task_list_link     String
+  work_packages      Work_Package[]
+  project_manager    User           @relation(fields: [project_manager_id], references: [user_id])
+  project_manager_id Int
+}
+
+model Work_Package {
+  wp_id        Int                  @id
+  wp_progress  Float
+  start_date   DateTime
+  duration     Int
+  budget       Int
+  dependencies String
+  deliverables String
+  rules        String
+  desc_bullets Description_Bullet[]
+  project      Project              @relation(fields: [project_id], references: [project_id])
```

Comment:
I was thinking about calling this parent_project and then something like parent_project_id or parent_proj_id but it just felt too long

Reply:
meh. project and project id is good